libcxx/include/__memory/allocator_traits.h

```diff
@@ -73,7 +73,7 @@ struct __void_pointer {
 template <class _Ptr, class _Alloc>
 struct __void_pointer<_Ptr, _Alloc, false> {
 #ifdef _LIBCPP_CXX03_LANG
-  using type _LIBCPP_NODEBUG = typename pointer_traits<_Ptr>::template rebind<void>::other;
+  using type = typename pointer_traits<_Ptr>::template rebind<void>::other;
```

Comment:
Why removed here?

Reply:
I noticed that the stage 1 generic-cxx03 builder failed, so I thought maybe Cxx03 didn't recognize _LIBCPP_NODEBUG?

Reply:
I think it must have failed due to `>>`

Reply:
OK, I've restored _LIBCPP_NODEBUG to all the CXX03 aliases.